src/axom/multimat/tests/multimat_test.cpp

```diff
@@ -328,7 +327,6 @@ void check_values(MultiMat& mm, std::string arr_name, MM_test_data<DataType>& da
           {
             EXPECT_NE(d, nullptr);
             EXPECT_EQ(*d, data.cellmat_dense_arr[dense_idx * data.stride + s]);
-            if(s == data.stride - 1) sparse_idx += 1;
```

Comment:
This one's confusing to me. Seems like we'd want to be using `sparse_idx` in this test and/or comparing data values via `sparse_idx` and `dense_idx`

Perhaps something changed when the MultiMat API was updated?

Thoughts @publixsubfan ?

Reply:
I think this might just be a leftover fragment from prior development; in the test code it looks like the source-of-truth for for both the dense and sparse cases is a flat dense array (`data.cellmat_dense_arr`), so we don't need the `sparse_idx` to access the reference data.